City-of-Helsinki/helsinki-paatokset
# PP-223: Cases integration

Added migration of cases from Ahjo API data + new content type for cases

To test:
- Run `make drush-cim && make drush-cr`
- Move `public/modules/custom/paatokset_ahjo_proxy/static/cases.json` to `public/sites/default/files`
- Run `make shell` and run `drush mim ahjo_cases`
- New cases should be imported correctly from the json file to content type `Case`
